src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -94,12 +94,12 @@ export default class CorrectNameRequest extends Mixins(NameRequestMixin) {
     v => this.validateNameRequestNumber(v) || 'Name Request Number is invalid'
   ]
   private entityPhoneNumberRules = [
-    v => this.isInputEntered(v, 'phone') || 'Phone number is required',
+    v => (v !== '' || !this.entityEmail) || 'Phone number is required',
```

Comment:
This looks backwards... please add a comment that it's one or the other, or something :)

Reply:
oh no your right, i can do this better. Will refactor!

Reply:
Ok, fixed my wonky rules. They didn't make any sense. Refactored and Simplified :P

Reply:
Does the new code still enforce "either phone number or email"?

Reply:
And do you have a unit test for this, so we find out right away if we break it in the future?

Reply:
Yes, the form itself will never be valid unless one or the other are present.
No, will add one!

Reply:
I can explain as well, this fix was more of a UX fix as the core of the validation wasn't broken prior. 

There is two layers of validation, the form itself and the input fields individually. Prior, the input fields were validating as if they were required, which technically neither is required, but one or the other needs to be there to complete ( validating at the form level ). So i removed the 'required rule' at the input level, and just relied on the form validation to check at least one of the values is present.